tldr/cross_file_calls.py

```
@@ -18,7 +18,7 @@
 import os
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import Iterator, Optional
+from typing import Any, Iterator, Optional
```

Ruff (0.14.14)

Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)